pywbem_mock/_mainprovider.py

```diff
@@ -1171,8 +1171,35 @@ def DeleteQualifier(self, namespace, QualifierName):
             :exc:`~pywbem.CIMError`: (CIM_ERR_INVALID_NAMESPACE)
             :exc:`~pywbem.CIMError`: (CIM_ERR_INVALID_PARAMETER)
             :exc:`~pywbem.CIMError`: (CIM_ERR_NOT_FOUND)
-        """
-
+            :exc:`~pywbem.CIMError`: (CIM_ERR_FAILED)
+
+        """
+        def qualifier_exists(cls, qualifier_name):
```

Comment:
I would prefer a terminology that the qualifier is used on the class component. Existence is something I would attribute to the qualifier declaration.

Reply:
qualifier_exiists_in_cls. DONE

Reply:
Not what I had in mind, but I see you like the "exists" terminology. Ok with me.